### PR TITLE
feat: TE7-T1022: Implement Create User Screen

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement Create User Screen

### CODE REVIEW

File: app/constants/Strings.ts [hunk 1]
```diff
@@ -52,5 +52,11 @@ export default {
     create: 'Create User',
     videos: 'Videos',
     profile: 'Your Profile'
-  }
+  },
+  defaultImg: 'https://cdn.vectorstock.com/i/preview-1x/77/30/default-avatar-profile-icon-grey-photo-placeholder-vector-17317730.jpg',
```
Comment (sagar-simformsolutions) [inline app/constants/Strings.ts]: use with dummy data

File: app/modules/create-user/CreateUserScreen.tsx [hunk 2]
```diff
@@ -1,7 +1,124 @@
...
+          </TouchableOpacity>
+        </View>
+      </View>
+      <KeyboardAvoidingView
+        style={styles.bottomContainer}
+        behavior={globalMetrics.isIos ? 'padding' : 'height'}
+        keyboardVerticalOffset={verticalScale(15)}>
+        <ScrollView showsVerticalScrollIndicator={false}>
+          <CustomInput
+            editable={true}
```
Comment (sagar-simformsolutions) [inline app/modules/create-user/CreateUserScreen.tsx]: its not required to pass true, for editable here, just mention editable

File: app/modules/create-user/CreateUserScreen.tsx [hunk 3]
```diff
@@ -1,7 +1,124 @@
...
+            name={Strings.formInputNames.email}
+            onChangeText={formik.handleChange(Strings.formInputNames.email)}
+            onSubmitEditing={() => handleSubmitEdit(formRefs, 0)}
+            returnKeyType="next"
+            defaultValue={formik.values.email}
+            touched={formik.touched.email}
+            error={formik.errors.email}
+          />
+          <CustomInput
+            editable={true}
```
Comment (sagar-simformsolutions) [inline app/modules/create-user/CreateUserScreen.tsx]: check here also, check others for same

**Review by sagar-simformsolutions — CHANGES_REQUESTED** [review]

File: app/constants/Strings.ts (hunk 1 above)
Comment (jigar-s-simform) [reply app/constants/Strings.ts]: resolved

File: app/modules/create-user/CreateUserScreen.tsx (hunk 2 above)
Comment (jigar-s-simform) [reply app/modules/create-user/CreateUserScreen.tsx]: resolved

File: app/modules/create-user/CreateUserScreen.tsx (hunk 3 above)
Comment (jigar-s-simform) [reply app/modules/create-user/CreateUserScreen.tsx]: resolved

**Review by sagar-simformsolutions — APPROVED** [review]

**Pull request merged**